bundles/org.openhab.binding.meross/README.md

```diff
@@ -0,0 +1,95 @@
+# Meross Binding
+
+This binding integrates **Meross**&reg; devices
+
+## Supported Things
+
+Supported thing types
+
+- bridge
+- light
+
+|   Meross Name       | Type   | Description         | Supported | Tested| 
+|---------------------|--------|---------------------|-----------|--------|
+| Smart ambient light | msl430 | Smart ambient light | yes       | yes    |
+| Smart plug          | mss210 | Smart plug          | yes       | yes    |
+
+Anyway, the togglex capability may consent to control on/off of the majority of plugs and bulbs
```

Comment:
Is this relevant? If it is, it might need some work or is a togglex capability clear to all Meross users?
```suggestion
```